bot.py

```diff
@@ -0,0 +1,91 @@
+import os
+import platform
+import contextlib
+import hashids
+import datetime
+from google.cloud import datastore
+
+import discord
+from discord.ext import commands
+from discord import option
+from dotenv import load_dotenv
+
+import scanner
+load_dotenv()
+DISCORD_TOKEN = os.getenv('DISCORD_TOKEN')
+SALT = os.getenv('SALT')
+
+intents = discord.Intents.default()
+bot = commands.Bot(intents=intents)
+
+hashid_client = hashids.Hashids(salt=SALT, min_length=6)
+
+def upload_to_datastore(result, discord_user_id=None):
+    print(f'Uploading to datastore...')
+
+    datastore_client = datastore.Client.from_service_account_json(
+        'catalog-scanner.json')
+
+    temp_key = datastore_client.key('Catalog')
+    key = datastore_client.allocate_ids(temp_key, 1)[0]
+    catalog = datastore.Entity(key, exclude_from_indexes=['data', 'unmatched'])
+
+    key_hash = hashid_client.encode(key.id)
+    catalog_data = '\n'.join(result.items).encode('utf-8')
+    catalog.update({
+        'hash': key_hash,
+        'data': catalog_data,
+        'locale': result.locale,
+        'type': result.mode.name.lower(),
+        'created': datetime.datetime.utcnow(),
+        'discord_user_id': discord_user_id
+    })
+    if result.unmatched:
+        unmatched_data = '\n'.join(result.unmatched).encode('utf-8')
+        catalog['unmatched'] = unmatched_data
+    
+    datastore_client.put(catalog)
+    return catalog
+
+async def handle_message(ctx: discord.ApplicationContext, attachment: discord.Attachment):
+	attch_type = attachment.content_type.split('/') # Attachment.content_type returns a {type}/{file_format} string
+	if attch_type[0] != 'video':
+		await ctx.respond('The attachment needs to be a valid video file', ephemeral=True)
+	else:
+		try:
```

Comment:
I think you want this try/catch only around `scanner.scan_media(temp_file)` instead of the whole thing? Since the error message talks specifically about scanning